CHANGELOG.md

```diff
@@ -15,7 +15,9 @@ The change log is available [on GitHub][2].
   Adding EDSL support for arrays of tables
 * [#149](https://github.com/kowainik/tomland/issues/149):
   Removing records syntax from `PrefixTree`.
-  
+* [#152](https://github.com/kowainik/tomland/issues/152):
+  Removing `mdimap`.
```

Comment:
This is a breaking change. So let's write it in style of `**Breaking change:**` and `Migration guide`.

Also, could you please update version here from `## 0.6.0` to `## Unreleased: 1.0.0`?